### DISCUSSION TITLE
What's the minimal package/css list to implement tuiScrollbar with virtual scroll?

### DISCUSSION THREAD

**Discussion by haoksam** [taiga-family/taiga-ui#5483] [Q&A] [answered]
I've installed **core/cdk/kit** packages, imported **_TuiScrollbarModule_** and **_ScrollingModule_** (from angular/cdk), added _**taiga-ui-theme.less**_.
Virtual scroll by itself is working, but the style of the scroll bar is like browser default (ugly). 
What package/module/styles am I missing?

**Comment by waterplea** [comment]
Take a look at this example:
https://taiga-ui.dev/components/scrollbar#virtual
Have you added the `tui-zero-scrollbar` class?

**Reply by haoksam** [reply, marked as answer]
Yes, the class is there, but...
Turns out in order to make things look good you have to put everything into `<tui-root></tui-root>` and add TuiRootModule in imports. 
Never skip **[Getting Started](https://taiga-ui.dev/getting-started)** section of documentation. >_<
My bad. Sorry.
